README.md

```diff
@@ -14,7 +14,7 @@ Everyone is encouraged to run a chain locally for development and testing purpos
 
 Entire process of running local chain is automated by our tooling. The only prerequisites are:
 - `docker` and `tmux` installed from your favorite package manager
-- `go 1.16` or newer installed and available in your `PATH`
+- `go 1.17` or newer installed and available in your `PATH`
```

Comment:
The version in the root go.mod is 1.18 but in the build 1.17, would it make sense to keep just one version?